documentation/import-production-data-local.md

```diff
@@ -0,0 +1,88 @@
+# Import production data from cloud firestore to the local firestore
+
+based on this [tutorial](https://medium.com/firebase-developers/how-to-import-production-data-from-cloud-firestore-to-the-local-emulator-e82ae1c6ed8) with a few differences for our use case
```

Comment:
Based on [this tutorial](https://medium.com/firebase-developers/how-to-import-production-data-from-cloud-firestore-to-the-local-emulator-e82ae1c6ed8) with a few differences for our use case.